types/react-with-styles/lib/ThemedStyleSheet.d.ts

```diff
@@ -9,6 +9,7 @@ interface ThemedStyleSheet {
   resolveLTR: () => any;
   resolveRTL: () => any;
   flush: () => void;
-}
+};
 
+export type ThemedStyleSheet = typeof ThemedStyleSheet;
 export default ThemedStyleSheet;
```

Comment:
Seems odd that this would be exported as a named and default export.
User of value would do: `import ThemedStyleSheet from '...';`
User of type would do: `import { ThemedStyleSheet } from '...';`

Should we not export the type in this case, and users of the type could do the `typeof ThemedStyleSheet`? 

Thinking longer-term, if this file were written in TS with JS & DTS emitted, it wouldn't get a type export like this anyways.

Reply:
Or if the type is commonly used enough we could export it under a different name

Reply:
This was done based on Ryan's [suggestion](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38842#issuecomment-538157377) for backward compatibility

Reply:
Ah. Good point. I missed that, thanks!